.changeset/dry-coins-juggle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@kadena/proof-of-us': patch
+'@kadena/react-ui': patch
+---
+
+Update the textfield and numberfield components to the new branding
```

Comment:
I would recommend adding more detail when writing changesets. Essentially add any information you would want someone using the library to know before they upgrade. So for example any breaking changes, and new features, etc. Otherwise someone might upgrade and not realize why something is suddenly not working